src/main/java/com/tileman/TilemanModePlugin.java

```diff
@@ -213,15 +225,70 @@ protected void shutDown() {
         points.clear();
     }
 
-    private Collection<GroundMarkerPoint> getPoints(int regionId) {
-        return getGroundMarkerConfiguration(REGION_PREFIX + regionId);
+    public void importGroundMarkerTiles() {
+        // Get and store all the Ground Markers Regions
+        List<String> groundMarkerRegions = removeRegionPrefixes(configManager.getConfigurationKeys("groundMarker.region"));
+        // If none, Exit function
+
+        // Get and store array list of existing Tileman World Regions (like updateTileCounter does)
+        List<String> tilemanModeRegions = removeRegionPrefixes(configManager.getConfigurationKeys(CONFIG_GROUP + ".region"));
+
+        // CONVERSION
+        // Loop through Ground Marker Regions
+        for (String region: groundMarkerRegions) {
+            // Get Ground Markers Region's Tiles
+            ArrayList<TilemanModeTile> groundMarkerTiles =
+                    new ArrayList<>(getConfiguration("groundMarker", REGION_PREFIX + region));
+            // If region already exists in Tileman World Regions Array:
+            if(tilemanModeRegions.contains(region)) {
+                // Create Empty ArrayList for Region;
+                // Get Tileman Region's tiles and add them to the region array list
+                ArrayList<TilemanModeTile> regionTiles = new ArrayList<>(getTiles(Integer.parseInt(region)));
+
+                // Create int for regionOriginalSize;
+                // Set regionOriginalSize to arraylists length
+                int regionOriginalSize = regionTiles.size();
+                int nonUniqueTiles = 0;
+
+                // Loop through Ground Markers Points
+                for(TilemanModeTile groundMarkerTile: groundMarkerTiles) {
+                    // If Ground Marker point already exists in Tileman World Region: Break loop iteration
+                    if(regionTiles.contains(groundMarkerTile)){
+                        nonUniqueTiles++;
+                        continue;
+                    }
+                    // Add point to array list
+                    regionTiles.add(groundMarkerTile);
+                }
+                // If regionOriginalSize != current size
+                if(regionOriginalSize != regionTiles.size()) {
+                    // Save points for arrayList
+                    savePoints(Integer.parseInt(region), regionTiles);
+                }
+            } else {
+                // Save points for that region
+                savePoints(Integer.parseInt(region), groundMarkerTiles);
+            }
+        }
+    }
+
+    private List<String> removeRegionPrefixes(List<String> regions) {
```

Comment:
I swear we already had a method to do this...

Reply:
Not a method, but we had something similar happening in the updateTileCounter method, making it use this functionality instead (added another method to remove region prefix for individual region strings)